testutil/tx/eth.go

```diff
@@ -104,6 +104,12 @@ func CreateEthTx(
 	fromAddr := common.BytesToAddress(from.Bytes())
 	chainID := evmconfig.GetChainConfig().ChainID
 
+	baseFeeRes, err := appEvmos.EvmKeeper.BaseFee(ctx, &evmtypes.QueryBaseFeeRequest{})
+	if err != nil {
+		return nil, err
```

Comment:
I would wrap this error with `"cosmossdk.io/errors"` to provide more info, like `failed during creation of eth tx in test`.